_data/experience.yml

```diff
@@ -1,16 +1,19 @@
 # Jobs
-# Surftocat
```

Comment:
This code seems unnessary. Is there something we can do to make it more applicable to the user?